config.example.json

```diff
@@ -1,3 +1,5 @@
 {
-  "example-argument": "example-value"
+  "oidc-base-url": "https://auth.philanthropydatacommons.org/",
```

Comment:
Maybe include the `realms/pdc` here too.

Reply:
Good call!